src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -30,21 +30,13 @@ const FormDefinitionDetailPage = () => {
 
   return (
     <>
-      <Stack gap={0.5} sx={{ my: 2 }}>
-        <Typography
-          variant='caption'
-          color='links'
-          sx={{ textTransform: 'uppercase', fontWeight: 800 }}
-        >
-          Selected Form
-        </Typography>
-        <Stack direction='row' gap={1}>
-          <Typography variant='h3'>
-            {formIdentifier.displayVersion.title}
-          </Typography>
-        </Stack>
-      </Stack>
       <Stack gap={2}>
+        <Typography variant='h3'>
```

Comment:
just noting here, for interest, though I don't think it should be fixed on this PR, that one of the recommendations coming out of the a11y audit for this app is that we should divorce heading hierarchy from heading appearance. For good screen reader nav, ideally, the first heading on a page is a h1 -- even if it's styled in a way that appears like an h3 in other parts of the app.

It's a "best practice" as opposed to a WCAG standard, though, so perhaps that makes it lower priority to fix!

Reply:
Well thats an easy fix, updated with https://github.com/greenriver/hmis-frontend/pull/802/commits/af570354984ab4da66eb62ef9214663930be4683. I'm not sure if it's correct to have the overline text span inside the h1 tag.